.circleci/config.yml

```diff
@@ -10,6 +10,10 @@ executors:
     docker:
       - image: aeternity/builder
         user: builder
+  builder_container_1804:
+    docker:
+      - image: aeternity/builder:1804
```

Comment:
I see there is [CI job pushing it](https://github.com/aeternity/docker-builder/blob/1b919eeac1628ea41eb2f525665148cad9e82862/.circleci/config.yml#L71). Ok.